extractor/text.go

```diff
@@ -473,28 +475,32 @@ func (to *textObject) setCharSpacing(x float64) {
 		return
 	}
 	to.state.tc = x
+	if verboseGeom {
+		common.Log.Info("setCharSpacing: %.2f state=%s", x, to.state.String())
+	}
 }
 
 // setFont "Tf". Set font.
 func (to *textObject) setFont(name string, size float64) error {
 	if to == nil {
 		return nil
 	}
+	to.state.tfs = size
 	font, err := to.getFont(name)
-	if err == nil {
-		to.state.tfont = font
-		if len(*to.fontStack) == 0 {
-			to.fontStack.push(font)
-		} else {
-			(*to.fontStack)[len(*to.fontStack)-1] = font
+	if err != nil {
+		if err == model.ErrFontNotSupported {
+			// TODO(peterwilliams97): Do we need to handle this case in a special way?
```

Comment:
if font is not supported, is there anything that makes sense to do?
Probably need to collect such cases and look at.